scripts/terraform-fmt.ts

```diff
@@ -0,0 +1,66 @@
+import { exec, ExecException } from 'child_process';
+import { exit } from 'process';
+
+const BINARY = 'terraform';
+const DIR = 'packages/airnode-deployer/terraform';
+const CMD_CHECK = `${BINARY} fmt -list=true -write=false -recursive ${DIR}`;
+const CMD_WRITE = `${BINARY} fmt -list=true -write=true -recursive ${DIR}`;
+
+if (process.argv.length !== 3) {
+  console.error('Wrong script usage!');
+  console.error('terraform-fmt.ts check | write');
+  exit(3);
```

Comment:
Is `exit(3)` right? Is it using these codes https://nodejs.org/api/process.html#process_exit_codes?

This also applies to the other places where `exit()` is called with a code of 2 or 3

Reply:
Oh, I didn't know about these. I just wanted to distinguish between formatting error (1), processing error (2) and CLI error (3). I'll try to accommodate them even though I don't know if it will map well to those in the documentation.

Reply:
Hmm, it looks like those are internal Node.js exit codes, I'm not sure those are intended to be used by an external code. I would rather suggest using codes let's say 21, 22, and 23 (higher than 13) so it's clear that those are codes returning from the script itself and not from the Node.js. What do you think, does that make more sense?

Reply:
I've changed the invalid argument to 22 but then I just followed with 23 and 24 as there were no appropriate codes in the errno list. And afaik that list is not commonly used, scripts and tools have usually their own error codes.

But we will avoid colliding with the internal Node.js error codes and at least one will be compatible with the Unix errno codes :man_shrugging: